cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -29,6 +29,20 @@
 namespace gluten {
 
 namespace {
+#define LOG_VALIDATION_MSG_FROM_EXCEPTION(err)                                                                                       \
+  logValidateMsg(fmt::format(                                                                                                        \
+      "Validation failed due to exception caught at file:{} line:{} function:{}, thrown from file:{} line:{} function:{} reason:{}", \
+      __FILE__,                                                                                                                      \
+      __LINE__,                                                                                                                      \
+      __FUNCTION__,                                                                                                                  \
+      err.file(),                                                                                                                    \
+      err.line(),                                                                                                                    \
+      err.function(),                                                                                                                \
+      err.message()));
+
+#define LOG_VALIDATION_MSG(reason) \
+  logValidateMsg(fmt::format(      \
+      "Validation failed at file:{}, line:{}, function:{}, reason:{}", __FILE__, __LINE__, __FUNCTION__, reason));
```

Comment:
I think it may be better to remove the trailing ";" and just append it on the caller side.

Reply:
ok.